backend/server.py

```
@@ -9,347 +9,195 @@
-    try:
-        return await health_manager.get_health_status()
-    except Exception as e:
-        raise HTTPException(status_code=500, detail=f"Failed to get service status: {e}")
\ No newline at end of file
+# Direct execution entry point
+if __name__ == "__main__":
+    import uvicorn
+    port = int(os.environ.get("PORT", 8010))
+    host = os.environ.get("HOST", "0.0.0.0")
+    logger.info(f"Starting server on {host}:{port}...")
+    uvicorn.run("server:app", host=host, port=port, reload=True)
```

Ruff (0.13.1)

No newline at end of file

Add trailing newline

(W292)